packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -465,6 +466,7 @@ export class Slider extends FormAssociatedSlider implements SliderConfiguration
             this.setupTrackConstraints();
             (e.target as HTMLElement).focus();
             window.addEventListener("mouseup", this.handleWindowMouseUp);
```

Comment:
I think we should be making sure these listeners are removed on disconnect.

Reply:
This is weird. It's a different PR that I have out. I'll resubmit a corrected PR.

Reply:
There were a lot of listeners, with listeners attaching other listeners. Rather than risk missing one, or a new one being added and by one of the methods, I created a 'remove' flag for each so that I could use the same methods to add and remove event listeners to ensure parity. It feels pretty heavy handed, but I feel like this way would be less prone to errors.